index.js

```diff
@@ -37,4 +47,4 @@ const stringify = (object, type) => {
     }
 }
 
-module.exports = { parse, stringify }
+module.exports = { parse, stringify, removeFileExt }
```

Comment:
true but how am I going to be able to test it if I can't directly access the function

see: https://github.com/MarkTiedemann/naml/pull/1/commits/38695a165bccd5f4b274eb2e0023acaeba0329e2#diff-f3b199869efde8696563e62cbae0bec6